cli/cmd/generate_support_archive.go

```diff
@@ -442,96 +442,96 @@ func getIngressGateway(keptnNamespace string) *errorableStringResult {
+	writeErrorableStringResult(newErrorableStringResult(keptnutils.ExecuteCommand("kubectl", []string{"get", "deployments", "-o", "wide", "-n", namespace})),
 		filepath.Join(dir, "deployments.txt"))
 }
 
 func writePods(namespace, dir string) {
 	fmt.Println("Retrieving list of pods in " + namespace)
-	writeErrorableStringResult(newErrorableStringResult(exechelper.ExecuteCommand("kubectl", "get pods -owide  -n "+namespace)),
+	writeErrorableStringResult(newErrorableStringResult(keptnutils.ExecuteCommand("kubectl", []string{"get", "pods", "-o", "wide", "-n", namespace})),
 		filepath.Join(dir, "pods.txt"))
 }
 
 func writeServices(namespace, dir string) {
 	fmt.Println("Retrieving list of services in " + namespace)
-	writeErrorableStringResult(newErrorableStringResult(exechelper.ExecuteCommand("kubectl", "get services -owide -n "+namespace)),
+	writeErrorableStringResult(newErrorableStringResult(keptnutils.ExecuteCommand("kubectl", []string{"get", "services", "-o", "wide", "-n", namespace})),
 		filepath.Join(dir, "services.txt"))
 }
 
 func writeVirtualServices(namespace, dir string) {
 	fmt.Println("Retrieving list of virtual services in " + namespace)
-	writeErrorableStringResult(newErrorableStringResult(exechelper.ExecuteCommand("kubectl", "get vs -owide -n "+namespace)),
+	writeErrorableStringResult(newErrorableStringResult(keptnutils.ExecuteCommand("kubectl", []string{"get", "vs", "-o", "wide", "-n", namespace})),
 		filepath.Join(dir, "virtualservices.txt"))
 
 }
 
 func writeIngresses(namespace, dir string) {
 	fmt.Println("Retrieving list of ingresses in " + namespace)
-	writeErrorableStringResult(newErrorableStringResult(exechelper.ExecuteCommand("kubectl", "get ingress -owide -n "+namespace)),
+	writeErrorableStringResult(newErrorableStringResult(keptnutils.ExecuteCommand("kubectl", []string{"get", "ingress", "-o", "wide", "-n", namespace})),
 		filepath.Join(dir, "ingresses.txt"))
 }
 
 func writePodDescriptions(namespace, dir string) {
 	fmt.Println("Retrieving pod descriptions in " + namespace)
-	res, err := exechelper.ExecuteCommand("kubectl",
-		`get pods --template '{{range .items}}{{.metadata.name}}{{"\n"}}{{end}}' -n `+namespace)
+	res, err := keptnutils.ExecuteCommand("kubectl",
+		[]string{"get", "pods", "--template", "{{range .items}}{{.metadata.name}}{{'\n'}}{{end}}", "-n", namespace})
 	if err != nil {
 		writeErrorableStringResult(newErrorableStringResult("", err),
 			filepath.Join(dir, "poddescriptions.txt"))
 		return
 	}
 	for _, pod := range strings.Split(strings.TrimSpace(res), "\n") {
-		res := newErrorableStringResult(exechelper.ExecuteCommand("kubectl",
-			"describe pod "+pod+" -n "+namespace))
+		res := newErrorableStringResult(keptnutils.ExecuteCommand("kubectl",
+			[]string{"describe", "pod", pod, "-n", namespace}))
 		writeErrorableStringResult(res, filepath.Join(dir, pod+"_description.txt"))
 	}
 }
```

Comment:
I think this is also not working.

`poddescriptions.txt` contains the following output:

```
Error executing command kubectl get pods --template {{range .items}}{{.metadata.name}}{{'
'}}{{end}} -n keptn: exit status 1
error: error parsing template {{range .items}}{{.metadata.name}}{{'
'}}{{end}}, template: output:1: unexpected unterminated character constant in command
```

I only see this in the global file poddescriptions.txt, but I don't see the individual pods.